.gitignore

```diff
@@ -17,3 +17,31 @@ api/tmp/*
 api/.byebug_history
 api/config/master.key
 api/.ruby-gemset
+
+
+
+
+# dependencies
+client/node_modules
+client/.pnp
+client/.pnp.js
+
+# testing
+client//coverage
+
+# production
+client//build
+
+# misc
+client/.DS_Store
+client/.env.local
+client/.env.development.local
+client/.env.test.local
+client/.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+client/yarn.lock
```

Comment:
I'm guessing these should all be prefixed with `client/` as we're going to run npm (or yarn) from inside the client folder.